clients/src/main/java/org/apache/kafka/common/requests/ProduceRequest.java

```diff
@@ -123,7 +123,7 @@ public int timeout() {
         return timeout;
     }
 
-    public Map<TopicPartition, MemoryRecords> partitionRecords() {
+    public Map<TopicPartition, MemoryLogBuffer> partitionRecords() {
```

Comment:
Ack. There are probably a few of these. I'll do another pass and try to find others.